sensor_msgs/Image img
---
DetectedObjectsArray detected
